### DISCUSSION TITLE
immune.combined Rename identities error

### DISCUSSION THREAD

**Discussion by GreyRockIQ** [satijalab/seurat#5448] [Q&A]
Hello
I am unable get rename identities error as below

immune.combined <- Seurat::RenameIdents(immune.combined, `0` = "CD14 Mono", `1` = "CD4 Naive T", `2` = "CD4 Memory T",
    `3` = "CD16 Mono", `4` = "B", `5` = "CD8 T", `6` = "NK", `7` = "T activated", `8` = "DC", `9` = "B Activated",
    `10` = "Mk", `11` = "pDC", `12` = "Eryth", `13` = "Mono/Mk Doublets", `14` = "HSPC")
Seurat::DimPlot(immune.combined, label = TRUE)

Error in RenameIdents.Seurat(immune.combined, `0` = "CD14 Mono", `1` = "CD4 Naive T", : 
Cannot find any of the provided identities

Any suggestions?
Thanks
GreyRock

**Comment by samuel-marsh** [comment]
What is the result of: `Idents(immune.combined)`?

**Comment by ikun-jinitaimei** [comment]
Just restart Rstudio and the problem will be solved.
.rs.restartR()
